CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ the code was deployed.
 
 ## [Unreleased]
```

Comment:
I'm a little concerned about this code being deployed to machines in production without at least a release tag.

If something funny happens on those machines, I'd want to have a way to look back and see the history of what was deployed. Do you know what commit you used to create the binary that is deployed at the clients who have this already? (Remember not to use their names on our public GitHub :wink: ). 

What do you think of a `10.8.0-beta` to indicate that this is a pre-release version? ([ref](https://semver.org/spec/v2.0.0.html))